mxcube3/routes/Login.py

```diff
@@ -101,6 +101,10 @@ def signout():
     mxcube.queue = qutils.new_queue()
     mxcube.shapes.clear_all()
 
+    if mxcube.CURRENTLY_MOUNTED_SAMPLE:
+        if mxcube.CURRENTLY_MOUNTED_SAMPLE_DATA['location'] == 'Manual':
```

Comment:
Where is  CURRENTLY_MOUNTED_SAMPLE_DATA defined the first time, and could it not be merged with CURRENTLY_MOUNTED_SAMPLE ?

Reply:
scutils Line 108, once a sample is mounted.. and yeah, I guess it can be merged... I will do that